libs/ui/src/templates/ArtClubTemplate/ArtClubTemplate.tsx

```diff
@@ -64,8 +75,40 @@ export const ArtClubTemplate: FC = () => {
     locale: locale as StrapiLocale,
   })
 
-  const createArt = (data: CreateArtFormFieldValues & { images: Blob[] }) => {
-    // mutate(data)
+  const createArtOnSuccess = () => {
+    createArtFormDisclosure.onClose()
+    createArtSuccessDisclosure.onOpen()
+  }
+
+  const createArtOnError = () => {
+    toast({
+      title: 'Error',
+      description: 'Something went wrong',
+      status: 'error',
+      duration: 5000,
+      isClosable: true,
+    })
+  }
+
+  const { mutate, isLoading: createArtIsLoading } = useCreateArt({
+    onSuccess: createArtOnSuccess,
+    onError: createArtOnError,
+  })
+
+  const createArt = async (
+    data: CreateArtFormFieldValues & { images: Blob[] },
+  ) => {
+    if (!auth.user) return
+
+    const slug = slugify(data.title)
+    const formBody = {
+      ...data,
+      slug,
+      artist: auth.user.id,
+      categories: data.categories?.map(c => Number(c.value)),
+    }
+
+    mutate(formBody)
```

Comment:
For an alternative solution, you can also use useMutation callbacks.

```suggestion
    mutate(formBody, {
      onSuccess: createArtOnSuccess,
      onError: createArtOnError
    })
```